patches/tModLoader/Terraria/Item.cs.patch

```diff
@@ -24,6 +24,7 @@
 +		}
 +
 +		internal GlobalItem[] globalItems = new GlobalItem[0];
++		internal GlobalItem[] globalItemsInstanced {  get { return globalItems.Where(gI => gI.InstancePerEntity).ToArray(); } }
```

Comment:
this is redundant, all globalItems on an item are instanced

Reply:
@Chicken-Bones jopo said not when I asked